src/elfpy/utils/data.py

```diff
@@ -18,23 +18,45 @@ def format_trades(analysis_dict):
     trades["fee_in_usd"] = trades.fee
     trades["fee_in_bps"] = trades.fee / trades.out_without_fee * 100 * 100
     base_asset_liquidity_usd = trades.base_asset_reserves * trades.base_asset_price
-    token_asset_liquidity_usd = (
-        trades.token_asset_reserves * trades.base_asset_price * trades.spot_price
-    )
+    token_asset_liquidity_usd = trades.token_asset_reserves * trades.base_asset_price * trades.spot_price
     trades["total_liquidity_usd"] = base_asset_liquidity_usd + token_asset_liquidity_usd
     trades["trade_volume_usd"] = trades.out_with_fee
     # calculate percent change in spot price since the first spot price (after first trade, kinda weird)
-    trades["price_rate_of_change"] = (
+    trades["price_total_return"] = (
         trades.loc[:, "spot_price"] / trades.loc[0, "spot_price"] - 1
```

Comment:
Did you verify that all of the example notebooks still run with this change?

Reply:
definitely not, but fee_analysis and rate_scenario do. i'll check now.

Reply:
ctrl-shift-f found 5 lines i hadn't manually changed. done now.